src/Apps/ArtQuiz/artQuizRoutes.tsx

```diff
@@ -1,8 +1,21 @@
 import loadable from "@loadable/component"
 import { RedirectException } from "found"
 import { graphql } from "react-relay"
+import { ArtsyResponse } from "Server/middleware/artsyExpress"
 import { AppRouteConfig } from "System/Router/Route"
 
+const LOGIN_COPY = "Log in to take the Art Quiz."
+const REDIRECT_URL = `/login?redirectTo=/art-quiz&copy=${LOGIN_COPY}`
```

Comment:
**nitpick [non-blocking]:** 

I see now that we're directing the user to login to take the art quiz via url. If they've already taken it and go straight to the results, I think this copy would be a little strange to read. Maybe we can make it more generalized?

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
I mean since the login is a pop-up modal this isn't a big deal, disregard!

Reply:
We can target that separately by redirecting on only the results route.

Reply:
(It's not a pop up actually)